cdap-docs/developers-manual/source/building-blocks/spark-programs.rst

```diff
@@ -4,17 +4,17 @@
 
 .. _spark:
 
-=============================================
-Spark Programs *(Beta, Standalone CDAP only)*
-=============================================
+==============
+Spark Programs
+==============
+
+.. rubric:: Beta, on Standalone and insecure Clusters only
 
 *Apache Spark* is used for in-memory cluster computing. It lets you load large sets of
 data into memory and query them repeatedly. This makes it suitable for both iterative and
 interactive programs. Similar to MapReduce, Spark can access **datasets** as both input
 and output. *Spark programs* in CDAP can be written in either Java or Scala.
 
-In the current release, Spark (version 1.0 or higher) is supported only in the Standalone CDAP. 
```

Comment:
Do we mention anywhere in the doc what Spark version(s) we support? We are using Spark 1.3 in Standalone and support 1.2, 1.3 and 1.4 on cluster.